src/scripts/choices.js

```diff
@@ -390,12 +390,15 @@ class Choices {
     return this._isSelectOneElement ? values[0] : values;
   }
 
-  setValue(args) {
+  /**
+   * @param {string[] | import('../../types/index').Choices.Item[]} items
+   */
+  setValue(items) {
```

Comment:
Would you mind updating the README too please :)

Reply:
Sorry, didn’t get this one - README already says that setValue parameter is always an array, here I’m removing unneeded spread+reconstruction of parameter and added parameters description to prevent that in future

Reply:
It just says "args" in the README though - it should be "items" - https://github.com/jshjohnson/Choices#setvalueargs